### ISSUE TITLE
Data Driven PHB Ranking - Sprint 23

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-data-tech#2450]
- Work on sidewalk criteria. Use the sidewalks layer from the ASMP Sidewalk Prioritization Map (5 pts - Very high, high; 3 pts - medium; 1 pt - low, very low; 0 pts - everything else).
- Work on speed limit criteria using Alan's new speed limit code data to see if we can make it work for PHB Ranking needs. If a segment has 2 speed limits, go with the higher speed limit.
- AMD wants to change pedestrian crashes to the last 5 years to align with Vision Zero. Andres sent me new data to process.

**Comment by jaime-mckeown** [comment]
- CSR field, 3 points CSR, 0 points No CSR.
- They would like to keep the criteria the same as before.
- Modify spreadsheet and script for updated points.

**Comment by jaime-mckeown** [comment]
- Used the Speed Limit City Code layer that's on MAINT as a basis for speed limits for PHB Ranking, coded all the speeds to the Street Public Trans HIN layer for coding.
- Many streets did not have a speed limit by the code as they fell under the prima facie for TX as 30 mph.
- I verified speed limits on various streets through Google Street View as they had not been updated in a long time in the code and the speeds had changed. Additionally, if one segment was different on a street, I verified whether or not it was correct.
- Modified the SPEED field part of the final script to use the new data to calculate the speed field.

**Comment by jaime-mckeown** [comment]
Sidewalk:
- Looked into the feature layer that was used in the Sidewalk Prioritization Map and discovered that it is pretty old and is a static layer. I informed AMD, we will discuss at next meeting.
- Moved forward with research on using the Sidewalk layer located on GISDM. I came up with a new method for analysis based on the street name so the sidewalk would only be counted if the street name matched since the sidewalk layer has a street name field. Unfortunately, the field is not always coded correctly as I found many areas where part of the sidewalk on one street was coded to the perpendicular street. This was the case for existing and absent sidewalks. 
- We will talk about this at tomorrow's meeting.
